pkg/webhook/pod/mutating/enhancedlivenessprobe_handler.go

```diff
@@ -0,0 +1,89 @@
+package mutating
+
+import (
+	"encoding/json"
+	"fmt"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"github.com/openkruise/kruise/pkg/features"
+	"github.com/openkruise/kruise/pkg/util"
+	utilfeature "github.com/openkruise/kruise/pkg/util/feature"
+)
+
+const (
```

Comment:
Consider moving the api to the common package like `./pkg/util` or `apis`?

Reply:
done